src/hooks/useMetric.js

```diff
@@ -7,22 +7,22 @@ import * as sinopiaMetrics from "../sinopiaMetrics"
 const useMetric = (name, params = null) => {
   const dispatch = useDispatch()
   const [metric, setMetric] = useState(null)
-  const [isMounted, setMounted] = useState(false)
+  const isMountedRef = useRef(false)
```

Comment:
This keeps the tests happier because it avoids actions after the test is completed.